client/sections.js

```diff
@@ -215,6 +215,11 @@ const sections = [
 		enableLoggedOut: true,
 		isomorphic: true,
 	},
+	{
+		name: 'jetpack-app',
+		paths: [ '/jetpack-app' ],
+		module: 'calypso/jetpack-app/',
```

Comment:
For consistency we could remove the forward slash.

```suggestion
		module: 'calypso/jetpack-app',
```